src/contracts/pods/EigenPodManager.sol

```diff
@@ -124,10 +106,22 @@ contract EigenPodManager is
                 existingDepositShares: existingDepositShares,
                 addedShares: addedShares
             });
-        } else if (sharesDelta < 0 && podOwnerDepositShares[podOwner] > 0) {
-            delegationManager.decreaseBeaconChainScalingFactor(
-                podOwner, uint256(podOwnerDepositShares[podOwner]), proportionOfOldBalance
-            );
+        } else if (sharesDelta < 0 && podOwnerDepositShares[podOwner] > 0) { // TODO >=
+            uint64 prevBeaconSlashingFactor = beaconChainSlashingFactor(podOwner);
+            uint64 newBeaconSlashingFactor = uint64(prevBeaconSlashingFactor.mulWad(wadSlashed));
+
+            emit BeaconChainSlashingFactorDecreased(podOwner, newBeaconSlashingFactor);
```

Comment:
could we add wadsSlashed here for indexing purposes

Reply:
[678abaf](https://github.com/Layr-Labs/eigenlayer-contracts/pull/892/commits/678abaf2c00c20254daf9b4515dca86cae2495d3)